Userland/Libraries/LibVT/Terminal.cpp

```diff
@@ -743,6 +746,9 @@ void Terminal::scroll_down(size_t count)
 // Insert `count` blank lines at the bottom of the region. Text moves up, top lines get added to the scrollback.
 void Terminal::scroll_up(u16 region_top, u16 region_bottom, size_t count)
 {
+if (region_top > region_bottom) {
+return;
+}
```

Comment:
This bit isn't necessary, let's keep the assert for more bugs

Reply:
it was for debugging, it wasnt suposed to be commited. you review too fast :D